frontend/src/metabase-lib/lib/Question.js

```diff
@@ -246,6 +246,25 @@ export default class Question {
     toUnderlyingData(): Question {
         return this.setDisplay("table");
     }
+
+    composeThisQuery(): ?Question {
```

Comment:
whoops sorry, it could be part of `Query` class interface then. updated the example